### PR TITLE
fix(ui): calculate service info correctly for networking.k8s.io/v1 ingress

### LINKED ISSUE #5839: Network view does not map Ingress with networking.k8s.io/v1 apiVersion to targeted services.
Checklist:

* [x] I've searched in the docs and FAQ for my answer: https://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

I recently updated my ingresses to use `apiVersion: networking.k8s.io/v1` due to the warnings that the `extensions/v1beta1` ingresses are deprecated.    However, now when I view my project's network view, the ingresses that I have updated do not show any relationship to the services.

**To Reproduce**

Create Ingress using the `networking.k8s.io/v1` apiversion to connect to a service (I used statefulset, but I think any deployment will show this issue).

**Expected behavior**

I would expect that the new ingress would show a mapping to a targetted service.

**Screenshots**

This is a screenshot of two ingresses using the new API version, and one using the old apiVersion.   Only the old apiVersion is showing a mapping to its targeted services.

Selection_065

**Version**

```shell
argocd: v2.0.0-rc1+0ca643f
  BuildDate: 2021-03-19T21:27:59Z
  GitCommit: 0ca643f027b99e8a5b931bb8ee9df42c3e4b64bf
  GitTreeState: clean
  GoVersion: go1.16
  Compiler: gc
  Platform: linux/amd64
```

**Logs**

```
Paste any relevant application logs here.
```

### PR DESCRIPTION
**Pull request by backjo**
Closes #5839

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [x] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [x] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [x] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [x] Does this PR require documentation updates?
* [x] I've updated documentation as required by this PR.
* [x] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [x] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [x] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: util/session/sessionmanager.go [hunk 1]
```diff
@@ -66,6 +66,11 @@ const (
 	autoRegenerateTokenDuration = time.Minute * 5
```
Comment (alexmt) [inline util/session/sessionmanager.go:66]: Looks like this change is not related to the PR. Can you revert please?

**Review by alexmt — DISMISSED** [review]
Thank you @backjo ! Changes look good. Can you please resolve linter error (resort imports) and revert change not related to the PR?

File: util/session/sessionmanager.go (hunk 1 above)
Comment (backjo) [reply util/session/sessionmanager.go:66]: yep

**Comment by backjo** [comment]
@alexmt should be good to go.

**Review by jessesuen — APPROVED** [review]
Change LGTM. One small nit

**Pull request merged**